polling_stations/templates/api_docs/blueprints/councils.apibp

```diff
@@ -7,49 +7,41 @@ Polling stations and districts are maintained by local councils so all of our da
 
 ## Councils Collection: JSON [/councils.json]
 
+
+### Response [GET]
```

Comment:
Naming an [action](https://github.com/apiaryio/api-blueprint/blob/master/Tutorial.md#actions) "Response" doesn't really make sense in the semantics of API Blueprint. It also has the side-effect when it comes to docs rendering that you're going to have loads of headers with the same anchor. i.e: at the moment I can deep link to https://wheredoivote.co.uk/api/#retrieve-a-council-geojson-4 but this will give you loads of different headings that all have the anchor `#response-4`.

If you wanted to just get rid of these headers completely, you can omit the action name and just use an HTTP verb for the action (i.e: make the header `### GET` for all of yours) and the action header will be omitted in the output. Given all your resources (endpoints) only have one action (there are no endpoints that accept a `PUT`, `PATCH`, `DELETE`, etc), that is another option and probably makes things shorter. The requests and responses are already individually labelled.

Reply:
@GeoWill are you able to look at this when you get in to updating the content of the docs? Doesn't have to be this PR, as the main point here is to add the CSS.

(Chris, Virginia can't install blueprint on her computer — or put another way, that version can't be installed on Macs at all any more — so me and Will are having to do all work that involves the docs)

Reply:
Yes for sure.